lib/handlers/quote/quote.ts

```diff
@@ -109,6 +110,9 @@ export class QuoteHandler extends APIGLambdaHandler<
           break
         case 500:
           metric.putMetric(`GET_QUOTE_500_CHAINID: ${chainId}`, 1, MetricLoggerUnit.Count)
+          if (useRpcGateway) {
```

Comment:
Im actually confused why you need to add the `RPC_GATEWAY` metrics, meanwhile the existing `GET_QUOTE` metrics can cover both rpc gateway launch vs not launched yet cases.

Reply:
The reason is that I want to only log the amount of requests that are processed via RPC gateway code path. It's useful to monitor a chain that's partially launched with RPC gateway, where `GET_QUOTE_xxx` will be strictly larger than `RPC_GATEWAY_GET_QUOTE_xxx`.